opencost-ui.yaml

```diff
@@ -41,12 +42,12 @@ subpackages:
     dependencies:
       provides:
         - nginx-config=0.${{package.full-version}}
```

Comment:
We need to remove this provides and keep the replaces

Reply:
Oh wait a moment... I could be wrong. Will follow up in the AM

Reply:
I can tell you this if I remove the provides its starting to install harbor-2.9-nginx-config during image building :D

Reply:
That's unfortunate and very weird behavior. I'll approve for now, but will be taking a closer look at this